po/fi.po

```diff
@@ -408,9 +408,9 @@ msgstr "ei pystytty lukemaan kansiota: {}"
 
 #: src/install.rs:778
 msgid "{} is a directory\n"
-"{} on kansio\n"
+"\n"
 ""
-msgstr ""
+msgstr "{} on kansio\n"
```

Comment:
You need to follow the exact formatting as above.

Reply:
My bad, sorry. Should be alright in the latest with identical formatting to msgid""

Reply:
Yes. I only commented on the first occurrence too. You need to do this elsewhere.

Reply:
Ah gotcha. I gave it a once over for formatting rigor.